qiskit/schemas/backend_configuration_schema.json

```diff
@@ -255,7 +255,26 @@
                                 "type": "array",
                                 "minItems": 0,
                                 "description": "A list of available parametric pulse shapes",
-                                "items": {"type": "string"}}
+                                "items": {"type": "string"}},
+                            "channels": {
+                                "type": "object",
+                                "minItems": 1,
+                                "patternProperties": {
+                                    "^[a-z0-9]+$": {
+                                        "type": "object",
+                                        "properties": {
+                                            "type": {"type": "string"},
+                                            "purpose": {"type": "string"},
+                                            "operates": {
+                                                "type": "object",
+                                                "properties": {
+                                                    "qubits": {"type": "array", "items": {"type": "integer", "minimum": 0}}
```

Comment:
Why `'operates': {'qubits': <int list>}` rather than `'operates': <int list>`?

Reply:
Discussed and resolved offline